packages/core/minos-microservice-networks/minos/networks/decorators/definitions/abc.py

```diff
@@ -30,6 +30,9 @@ class EnrouteDecorator(ABC):
     # noinspection PyFinal
     KIND: Final[EnrouteDecoratorKind]
 
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
```

Comment:
Could you add a test to ensure that additional named arguments will be stored on `kwargs` variable?